source/_default_sidebar.md.erb

```diff
@@ -14,15 +14,15 @@ Older [meetings](/meetings/) \| [nights](/nights/) \| [podcasts](/podcasts/)
 
 <%= meeting_calendar_link %>
 
-### Get In Touch
+### Community
 
 * [Mailing List](http://lrug.org/mailing-list)
 * [IRC](/irc/) (#lrug on freenode)
 * [El Rug (Twitter)](http://twitter.com/lrug)
 * Photos: [group (preferred)](http://flickr.com/groups/680991@N25/) or [tag](http://flickr.com/photos/tags/lrug/)
 * [Github](http://github.com/lrug)
-* [Last.FM Group](http://www.last.fm/group/LRUG)
-* [LinkedIn Group](http://www.linkedin.com/e/gis/117315/0027271A7866) (this links to a join request)
+* [LinkedIn Group](https://www.linkedin.com/groups/117315/) (this links to a join request)
```

Comment:
```suggestion
* [LinkedIn Group](https://www.linkedin.com/groups/117315/)
```
It now goes direct to the group so the extra text isn't needed.  Thanks for checking!